client/resources/aria-at-test-io-format.mjs

```diff
@@ -1653,7 +1653,7 @@ const UnexpectedBehaviorImpactMap = createEnumMap({
 const CommandSupportJSONMap = createEnumMap({
   FULL: 'FULL',
   FAILING: 'FAILING',
-  ALL_REQUIRED: 'ALL REQUIRED',
+  ALL_REQUIRED: 'ALL MUST',
```

Comment:
Same with this, but if it were to change, the enum name may as well be changed too